app/interactors/admin/contact_publisher.rb

```diff
@@ -0,0 +1,12 @@
+module Admin
+  class ContactPublisher
+    def initialize(contact)
+      @contact = contact
+    end
+
+    def call
```

Comment:
Existing interactors have more descriptive names for what they are and what their `invocation` method does.  Could we follow that pattern here as well?  While `ContactPublisher.new.call` is ok, `Rummager.new.call` is fairly non-descriptive.